File: PizzaShop/Shop.UI/wwwroot/js/admin/pizza.js
var app = new Vue({
    el: '#app',
    data: {
        loading: false,
        pizzas: [],
        pizzaModel: {
            id: 0,
            pizzaName: "AVOCADO VEG",
            pizzaDescription: "",
            pizzaPrice: 15.55,
            pizzaEnergy: 4432,
            pizzaSmallImagePath: ""
        },
       
    },
    mounted() {
        this.getAllPizzas();
    },
    methods: {
        getAllPizzas() {
            this.loading = true;
            axios.get('/Admin/pizza')
                .then(res => {
                    console.log(res);
                    this.pizzas = res.data;
                })
                .catch(err => {
                    console.log(err);
                })
                .then(() => {
                    this.loading = false;
                });
        },
        
        
    }
})

File: PizzaShop/Shop.Domain/Models/Pizza.cs
﻿using System.Collections.Generic;

namespace Shop.Domain.Models
{
    public class Pizza
    {
        public int Id { get; set; }
        public string PizzaName { get; set; }
        public string PizzaDescription { get; set; }
        public string PizzaSmallImagePath { get; set; }
        public string PizzaLargeImagePath { get; set; }
        public decimal PizzaPrice { get; set; }
        public double PizzaEnergy { get; set; }

        public List<PizzaTopping> PizzaToppings { get; set; }
    }
}